### DISCUSSION TITLE
graphql/codegen/plugins/typescript/typescript-apollo-next

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/graphql-code-generator#9346] [Docs Discussions]
# graphql/codegen/plugins/typescript/typescript-apollo-next

Apollo Client plugin for GraphQL Code Generator. Generates React Apollo HOC and React Hooks.

https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-apollo-next

**Comment by NilsNathorst** [comment]
I think there's a typo in the `withHooks` & `withHOC` sections. 

```diff
-  withHooks: 'I'
+  withHooks: true
```

```diff
-  withHOC: 'I'
+  withHOC: true
```

**Reply by ardatan** [reply]
Could you create a PR to fix it?

**Comment by Californian** [comment]
Is this plugin still maintained? If not, should it be removed?
